hemera/common/enumeration/entity_type.py

```diff
@@ -125,6 +125,10 @@ def entity_filter_mode(entity_types):
         return False
 
 
+ALL_ENTITY_COLLECTIONS = EntityType.__members__.keys()
```

Comment:
Where do you use these？

Reply:
'ALL_ENTITY_COLLECTIONS' will be used in the reorg module.
This modification here is to fix the compatibility errors caused by the refactoring, The actual logic repairs should be submitted separately.

Reply:
'ALL_ENTITY_COLLECTIONS' has been removed in the latest commit. 
Now the value will be obtained using DynamicEntityTypeRegistry.